source/features/parse-backticks.tsx

```diff
@@ -25,6 +25,9 @@ function init(): void {
 		'.Box-row.js-navigation-item a.link-gray-dark', // `isGlobalDiscussionList`
 		'[data-channel^="check_suites"] a', // `isActions`
 		'.repository-content .pr-toolbar h2', // `isActions` run
+		'.Details[data-issue-and-pr-hovercards-enabled] .Details-content--hidden a:first-child', // `isRepoRoot`
+		'.Details[data-issue-and-pr-hovercards-enabled] .Details-content--hidden pre', // `isRepoRoot`
+		'.js-issue-title', // `isIssue`
```

Comment:
Lets leave the "Repository refresh" layout so this could be cleaned up later
```suggestion
		'.Details[data-issue-and-pr-hovercards-enabled] .Details-content--hidden a:first-child', // `isRepoRoot` "Repository refresh" layout
		'.Details[data-issue-and-pr-hovercards-enabled] .Details-content--hidden pre', // `isRepoRoot` "Repository refresh" layout
		'.js-issue-title', // `isIssue` "Repository refresh" layout
```

Reply:
😄 I just added back them before you commented